moviepy/video/VideoClip.py

```diff
@@ -264,6 +269,13 @@ def write_videofile(
         verbose (deprecated, kept for compatibility)
           Formerly used for toggling messages on/off. Use logger=None now.
 
+        with_mask
+          If video has alpha channel, and you want to keep it after processing, this parameter must be True
+          You can use any FFMPEG encoding format that supports alpha channels.
+          Examples are 'png、qtrle' for '.mov'
```

Comment:
wouldn't be cool to warn if `with_mask` is true but the format doesn't support it?

Reply:
if  format doesn't support, for example:

clip = VideoFileClip("myvideo.mov", has_mask=True).subclip(100,120)
clip.write_videofile("my_new_video.mp4", codec='qtrle', with_mask=True)  #The result will fail to execute, FFMPEG will throw an exception: "codec not currently supported in container", because 'qtrle' not support '.mp4'. 

but this:
clip.write_videofile("my_new_video.mp4", codec='libx264', with_mask=True) #The result will be executed, but will not meet expectations, because 'libx264' not support alpha channel, the video will lose the alpha channel.

This is almost an inevitable problem, and for this reason I have detailed this in the code comments.

Reply:
Is this line supposed to read `Examples are 'png', 'qtrle' for '.mov'`? Your version has some strange comma character.

Reply:
Yes, here are some encoding formats that support the alpha channel:
'png' -->mov/mp4/mkv/...
'qtrle' -->mov
'vp6a' -->flv/f4v
'vp8/vp9' -->webm
etc.

Reply:
> Your version has some strange comma character.
I have fixed it.